ui/src/views/network/IngressEgressRuleConfigure.vue

```diff
@@ -293,12 +293,10 @@ export default {
             : response.authorizesecuritygroupegressresponse.jobid,
           successMessage: this.$t('message.success.add.rule'),
           successMethod: () => {
-            this.parentFetchData()
```

Comment:
`this.parentFetchData()` is needed here and in the subsequent methods sine it has to reload the page when a rule is added / deleted

Reply:
@davidjumani Not necessary because to avoid repeated `fetchData()` event I just allow utils/plugins.js which will send event to autogenView to execute it after job finish